railties/test/application/configuration_test.rb

```diff
@@ -3974,6 +3974,51 @@ def new(app); self; end
       assert_equal ActiveSupport::Cache::Coders::Rails61Coder, Rails.cache.instance_variable_get(:@coder)
     end
 
+    test "adds a time zone aware type if using PostgreSQL" do
+      original_configurations = ActiveRecord::Base.configurations
+      ActiveRecord::Base.configurations = { production: { db1: { adapter: "postgresql" } } }
```

Comment:
Setting `production:` for ActiveRecord::Base.configurations and creating `app "development"` looks inconsistent.